### ISSUE TITLE
Dispatching LOAD action on Safari crashes app

### ISSUE THREAD

**Issue by renanbandeira** [guns2410/redux-storage#8]
Related libraries used:
    "react-redux": "^5.0.6",
    "recompose": "^0.26.0",
    "redux": "^3.7.1",
    "redux-catch": "1.3.1",
    "redux-devtools-extension": "2.13.2",
    "redux-form": "6.8.0",
    "redux-remotedev": "0.3.0",
    "redux-saga": "^0.16.0",
    "redux-saga-actions": "1.0.6",
    "redux-storage": "4.1.2",
    "redux-storage-decorator-debounce": "github:whortaneto/redux-storage-decorator-debounce#pre_built",
    "redux-storage-decorator-immutable-filter": "1.0.1",
    "redux-storage-decorator-migrate": "1.1.0",
    "redux-storage-engine-localforage": "^2.0.0",
    "redux-storage-merger-simple": "1.0.5",
    "redux-thunk": "2.2.0"

I don't know why exactly, but when I open my app on Safari it doesn't load. After a lot of time of investigation, I realized that when the app trie to dispatch the LOAD action with all the state on payload, it crashes on Safari. The interesting thing is that if I rename the action type to any other name but REDUX_STORAGE_LOAD, it loads the app.

Has anybody faced this problem before?

**Comment by ptomasroos** [comment]
Can you please post a example repo of this bug in order to help out?

**Comment by renanbandeira** [comment]
Hi @ptomasroos. I was creating an example and it worked. I spent two weeks looking for it and yesterday I realized that the problem was because there was a saga watching for the LOAD action and coming into an infinite loop. Curiously this loop was silent on Chrome, but not on Safari. So, if I removed . the LOAD dispatch action, the saga was never being executed and the infinite loop was never being caught, making me think that Load action was causing the problem. Anyways, thanks for being available to help and sorry for dispatching a false alarm.
